vocables.js

```diff
@@ -91,6 +91,137 @@ const vocables = (german, norsk) => [
+  { [german]: ['oft'], [norsk]: ['ofte'] },
+  { [german]: ['selten'], [norsk]: ['sjelden'] },
+  { [german]: ['ein Garnelenbrot', 'ein Krabbenbrot'], [norsk]: ['et rekesmørbrød'] },
+  { [german]: ['etwas kosten', 'etwas probieren'], [norsk]: ['smake på noe'] },
+  { [german]: ['eine Sorte'], [norsk]: ['en sort'] },
+  { [german]: ['eine Biersorte'], [norsk]: ['en ølsort'] },
+  { [german]: ['sich hinsetzen', 'Platz nehmen'], [norsk]: ['sitte ned'] },
+  { [german]: ['unterschiedlich'], [norsk]: ['ulik', 'forskjellig'] },
+  { [german]: ['genießen'], [norsk]: ['nyte'] },
+  { [german]: ['ein Brötchen'], [norsk]: ['et rundstykke'] },
+  { [german]: ['später'], [norsk]: ['senere'] },
+  { [german]: ['früher', 'vorher'], [norsk]: ['før'] },
+  { [german]: ['sagen'], [norsk]: ['si'] },
+  { [german]: ['eine Sitte'], [norsk]: ['en skikk'] },
+  { [german]: ['Sitten und Bräuche'], [norsk]: ['skikk og bruk'] },
+  { [german]: ['ein Schinkenbrot'], [norsk]: ['et skinkesmørbrød'] },
+  { [german]: ['lieb', 'nett'], [norsk]: ['snill'] },
+  { [german]: ['bitte sei so gut'], [norsk]: ['vær så snill'] },
+  { [german]: ['essen'], [norsk]: ['spise'] },
+  { [german]: ['hungrig'], [norsk]: ['sulten'] },
+  { [german]: ['Hunger haben'], [norsk]: ['være sulten'] },
+  { [german]: ['so'], [norsk]: ['så'] },
+  { [german]: ['nehmen'], [norsk]: ['ta'] },
+  { [german]: ['danke'], [norsk]: ['takk'] },
+  { [german]: ['Zeit'], [norsk]: ['ei tid'] },
+  { [german]: ['durstig'], [norsk]: ['tørst'] },
+  { [german]: ['Durst haben'], [norsk]: ['være tørst'] },
+  { [german]: ['Entschuldigung!'], [norsk]: ['Unnskyld!'] },
+  { [german]: ['üblich', 'normal'], [norsk]: ['vanlig'] },
+  { [german]: ['normalerweise'], [norsk]: ['vanligvis'] },
+  { [german]: ['warm'], [norsk]: ['varm'] },
+  { [german]: ['am schlechtesten'], [norsk]: ['verst'] },
+  { [german]: ['nicht schlecht'], [norsk]: ['ikke så verst'] },
+  { [german]: ['wissen'], [norsk]: ['vite'] },
+  { [german]: ['er weiß'], [norsk]: ['han vet'] },
+  { [german]: ['ein Bier'], [norsk]: ['et øl'] },
+  { [german]: ['achso'], [norsk]: ['å ja'] },
+  { [german]: ['guten Morgen'], [norsk]: ['god morgen'] },
+  { [german]: ['guten Tag'], [norsk]: ['god dag'] },
+  { [german]: ['guten Abend'], [norsk]: ['god kveld'] },
+  { [german]: ['heute Abend'], [norsk]: ['i kveld'] }, 
+  { [german]: ['gute Nacht'], [norsk]: ['god natt'] },
+  { [german]: ['mach es gut'], [norsk]: ['ha det bra'] },
+
+  // From here sentences to practise (Lection 2)
+  { [german]: ['Das ist hier so üblich'], [norsk]: ['Det er skikk og bruk'] },
+  { [german]: ['Der Unterschied zwischen A und B'], [norsk]: ['Forskjellen på A og B'] },
+  { [german]: ['Han bruker å komme tidlig'], [norsk]: ['Er kommt normalerweise früh', 'Er kommt üblicherweise früh'] },
+  { [german]: ['In Norwegen ist man weniger formal als in Deutschland'], [norsk]: ['I Norge er man mindre formell enn i Tyskland'] },
```

Comment:
Wenn die Zeilen zu lang werden, kannst du die Sachen ruhig in mehrere Zeilen umbrechen (musst du ja nicht jetzt machen):

```js
{
  [german]: [
    'bla',
    'blo',
  ],
  [norsk]: [
    'bli',
    'blu',
  ],
},
```